### ISSUE TITLE
wish: document what the features are

### ISSUE THREAD

**Issue by markstos** [ljurk/jiramenu#2]

**Comment by ljurk** [comment]
good point, I didn't expect many people to find and use my tool. I can't really estimate when I can document the features, because I have some other projects, that I'm working on.
